ios/brave-ios/Sources/Brave/Frontend/Browser/New Tab Page/Backgrounds/NewTabPageBackgroundButtonsView.swift

```diff
@@ -142,12 +151,34 @@ class NewTabPageBackgroundButtonsView: UIView, PreferencesObserver {
         $0.centerX.equalToSuperview()
       }
     }
+
+    playButton.snp.remakeConstraints {
+      $0.centerX.equalToSuperview()
+      $0.centerY.equalToSuperview().offset(20)
+    }
+    // Hide the play button if the video is in landscape mode on a phone
+    if isLandscape && UIDevice.isPhone {
+      playButton.isHidden = true
+    } else {
+      playButton.isHidden = !playButtonAdded
+    }
   }
 
   @objc private func tappedButton(_ sender: UIControl) {
     tappedActiveButton?(sender)
   }
 
+  @objc private func tappedVideoPlayButton(_ sender: UIControl) {
+    tappedPlayButton?()
+  }
+
+  func addPlayButton() {
```

Comment:
This function does not add the button, lets think of a name that reflects the logic. Thanks